trajectory_tracker/test/test/trajectory_tracker_rostest.test

```diff
@@ -2,6 +2,7 @@
 <launch>
   <arg name="odom_delay" default="0.0" />
   <arg name="use_odom" default="false" />
+  <arg name="use_time_optimal_control" default="true" />
 
   <env name="GCOV_PREFIX" value="/tmp/gcov/trajectory_tracker_d$(arg odom_delay)_$(arg use_odom)" />
```

Comment:
Add `$(arg use_time_optimal_control)` to GCOV_PREFIX to avoid overwriting coverage profiles.

Reply:
Done